src/main/java/frc/robot/subsystems/drivetrain/DriveTrain.java

```diff
@@ -0,0 +1,18 @@
+package frc.robot.subsystems.drivetrain;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class DriveTrain extends SubsystemBase {
+
+    double xPower;
+    double yPower;
+    double angularPower;
+
+    public void updateDrivePowers(double forwardComponent, double sideComponent, double angularPower){
+        // xPower = forwardComponent;
+        // yPower = sideComponent;
+        // this.angularPower = angularPower;
+        // System.out.println("dt received drive powers");
```

Comment:
because we don't need (and in fact, don't want) to define a default `updateDrivePowers` method, this class can either be an `abstract class` or an `interface`.